src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/ClaimantResponseCuiCallbackHandler.java

```diff
@@ -126,6 +129,11 @@ private CallbackResponse aboutToSubmit(CallbackParams callbackParams) {
         UnavailabilityDatesUtils.rollUpUnavailabilityDatesForApplicant(
             builder, featureToggleService.isUpdateContactDetailsEnabled());
 
+        if (JudgmentAdmissionUtils.getLIPJudgmentAdmission(caseData) && featureToggleService.isJudgmentOnlineLive()) {
```

Comment:
A general comment - check if feature flag is live first, one less condition to check if its not on.

Reply:
yes, it's fixed